js-exercises/balanced-braces/balancedBraces.js

```diff
@@ -0,0 +1,34 @@
+function balancedBraces (str) {
+  let stack = [];
+
+    let openBracesArray = {
+        '{': '}',
+        '[': ']',
+        '(': ')'
+    };
+
+    let closedBracesArray = {
+        '}': true,
+        ']': true,
+        ')': true
+    }
+
+    for (let i = 0; i < str.length; i++) {
```

Comment:
Why not for-of loop?

Reply:
Fixed this comment and committed code. Used For -in loop to satisfy the code requirement.